src/runtime/propensity-server.js

```diff
@@ -39,6 +48,16 @@ export class PropensityServer {
     this.xhr_ = new Xhr(win);
     /** @private @const {number} */
     this.version_ = 1;
+
+    eventManager.registerEventListener(event => this.handleClientEvent_(event));
```

Comment:
We need to bind the callback to this instance of the class. 

eventManager.registerEventListener(this.handleClientEvent_.bind(this));

Reply:
It seemed to work either way, I wasn't sure which was preferable.  I've changed it back to a bind